README.md

```diff
@@ -172,6 +172,7 @@ export declare function stripComments(text: string, replaceCh?: string): string;
 export declare function getLocation(text: string, position: number): Location;
 
 export declare type Segment = string | number;
+export declare type JSONPath = Segment[];
```

Comment:
Not completely sure if `declare` is needed here; the README does not seem to use it consistently.